frontend/static/js/components/requestCard.js

```diff
@@ -1,32 +1,36 @@
-const requestOptions = () => {
+const requestOptions = (user, acceptRequest, declineRequest) => {
```

Comment:
Instead of individually adding Eventlisteners to each card, we could work with delegated eventlisteners on the scrollers or the overall page (see index.js). Once we have remove eventlisteners in place, it would make live much easier, as we wouldn't need to loop through all elements individually. 

Doesn't have to be done now. If anyone's interested one can look into `delegated eventlisteners` and `event bubbling` :D